webapp/src/cloud.ts

```diff
@@ -297,15 +299,15 @@ async function resolveConflict(local: Header, remoteFile: File) {
             }
         }
     } catch (e) {
-        // we want to swallow this and keep going since step 3. is the essentail one to resolve the conflcit.
+        // we want to swallow this and keep going since step 3. is the essential one to resolve the conflcit.
```

Comment:
small, since we're doing spelling :)
```suggestion
        // we want to swallow this and keep going since step 3. is the essential one to resolve the conflict.
```